src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -45,8 +50,11 @@
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@SuppressWarnings("PMD.TooManyMethods")
 public final class HtTimedWireTest {
 
+    private static final String UNUSED = "UNUSED";
+    private static final String TIMEOUT = "Did not failed after timeout";
```

Comment:
@aivinog1 please remove these two constants:

* both these constants introduce hidden coupling between tests
* if `UNUSED` were really unused, it would not be needed
* `TIMEOUT` has the wrong message structure - it should affirm the behavior being tested. Eg. if you're asserting that it throws a TimedOutException, the assertion's message would be "fails with a timedoutexception"

Reply:
I agree, thanks!